Source/WebCore/css/html.css

```diff
@@ -100,6 +100,8 @@ center {
 
 hr {
     display: block;
+    overflow: hidden;
```

Comment:
Is this necessary? When can the contents of an `<hr>` ever overflow? This incurs the creation of RenderLayers which is extra overhead.

Reply:
This is to affect the behavior with floats without a custom check in RenderBox.cpp

Rationale is at: https://github.com/whatwg/html/issues/2715#issuecomment-304881067